Jenkinsfile_CNP

```diff
@@ -0,0 +1,34 @@
+#!groovy
+
+properties([
+    [
+        $class: 'GithubProjectProperty',
+        displayName: 'Send letter service',
+        projectUrlStr: 'https://github.com/hmcts/send-letter-service'
+    ],
+    pipelineTriggers([
+        [$class: 'GitHubPushTrigger']
+    ])
+])
```

Comment:
the properties section is no longer required

Reply:
ok will remove this block.